https://lists.gnu.org/archive/html/bug-m4/2022-12/msg00000.html
https://lists.gnu.org/archive/html/bug-m4/2022-12/txtmGJxjX3osy.txt

Fix a sporadic failure of the test-posix_spawn-script test.

From a3efddb96f5f121b8a5bb1310dc82407546fd255 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Thu, 22 Dec 2022 21:19:34 -0800
Subject: [PATCH] posix_spawnp-tests: fix filename typo

Problem reported for GNU m4 by Mitchell Dorrell in:
https://lists.gnu.org/r/bug-m4/2022-12/msg00000.html
* tests/test-posix_spawnp-script.c (DATA_FILENAME):
Fix typo in file name that caused race with
test-posix_spawn-script.c.
---
 tests/test-posix_spawnp-script.c | 2 +-
 2 files changed, 10 insertions(+), 1 deletion(-)

diff --git a/tests/test-posix_spawnp-script.c b/tests/test-posix_spawnp-script.c
index 1e5fc5c110..d2035ed63e 100644
--- a/tests/test-posix_spawnp-script.c
+++ b/tests/test-posix_spawnp-script.c
@@ -28,7 +28,7 @@
 
 #include "macros.h"
 
-#define DATA_FILENAME "test-posix_spawn-script.tmp"
+#define DATA_FILENAME "test-posix_spawnp-script.tmp"
 
 int
 main ()
-- 
2.25.1
